src/ansys/units/systems.py

```diff
@@ -60,17 +64,24 @@ def __init__(
 
     def convert(self, quantity: ansunits.Quantity) -> ansunits.Quantity:
         """
-        Perform unit system conversions.
+        Convert a quantity into the unit system.
 
         Parameters
         ----------
         quantity : Quantity
-            Desired quantity object to convert.
+            Quantity to convert.
 
         Returns
         -------
         Quantity
-            Quantity object converted to the desired unit system.
+            Quantity object converted into the unit system.
+
+        Examples
+        --------
+        >>> ur = UnitRegistry()
+        >>> speed_si = Quantity(value=5, units= ur.m / ur.s)
+        >>> bt = UnitSystem(unit_sys="BT")
```

Comment:
I had not noticed `unit_sys` previously. My general policy is that if any name is consisdered too long (tenuous here) then I try to see what whole words can be reatined without losing meaning within the context. In this case `system` would work fine if `unit_sys` is considered too long.